assets/docs/sources/tutorial/13_export_and_feature.md

```
@@ -111,7 +112,7 @@ download_photo('456', option, extra=Feature.export_pdf)
 ├── [JM{Pid}]章节标题.pdf       ← 该章节导出为 1 个 PDF
 ```
 
-> 💡 **提示**：同一个 Feature，通过 `download_album` 和 `download_photo` 调用时会自动适配不同的导出行为，详见下方 [智能适配规则](#25-智能适配规则)。
+> 💡 **提示**：同一个 Feature，通过 `download_album` 和 `download_photo` 调用时会自动适配不同的导出行为，详见下方 [智能适配规则](#25)。
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)